### PR TITLE
Update RESPONSIBILITIES.md

### PR DESCRIPTION
**Pull request by varun-lodaya**
This change removes Amazon Security and makes OpenSearch Security team responsible for security of the project

### Description
This change replaces Amazon Security with OpenSearch Security that is responsible for security of the project.

### Issues Resolved
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by varun-lodaya** [comment]
This PR addresses the GH issue -https://github.com/opensearch-project/.github/issues/219

File: RESPONSIBILITIES.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Model the behavior set forward by the [Code of Conduct](CODE_OF_CONDUCT.md) and
 
 Security is your number one priority. Maintainer's Github keys must be password protected securely and any reported security vulnerabilities are addressed before features or bugs.
 
-Note that this repository is monitored and supported 24/7 by Amazon Security, see [Reporting a Vulnerability](SECURITY.md) for details.
+Note that this repository is monitored and OpenSearch Security team, see [Reporting a Vulnerability](SECURITY.md) for details.
```
Comment (peternied) [inline RESPONSIBILITIES.md:48]: OpenSearch Security Team is a proper noun per https://github.com/opensearch-project/.github/blob/main/SECURITY.md

```suggestion
Note that this repository is monitored and OpenSearch Security Team, see [Reporting a Vulnerability](SECURITY.md) for details.
```

**Review by peternied — COMMENTED** [review]
Thanks for making this change.

As written #219 is focused on the transition from Amazon -> OpenSearch Software Foundation; while this change is related to the transition I don't see this change resolving that issue.

File: RESPONSIBILITIES.md (hunk 1 above)
Comment (varun-lodaya) [reply RESPONSIBILITIES.md:48]: Yes, let me fix this and update the PR request.

**Comment by varun-lodaya** [comment]
> Thanks for making this change.
> 
> As written #219 is focused on the transition from Amazon -> OpenSearch Software Foundation; while this change is related to the transition I don't see this change resolving that issue.

Yeah, agreed, it does not completely close the GH issue, will send subsequent PRs to close the GH issue.

**Review by dblock — APPROVED** [review]

**Comment by dblock** [comment]
+1 on removing 1 responsibility that Amazon has, looking forward to the other places as @peternied mentioned

**Comment by varun-lodaya** [comment]
Can somebody help merge this?

**Comment by dblock** [comment]
> Can somebody help merge this?

It needs to sit in here till bake time is done. It's a mechanism to leave things open for comment for 48 hours.

**Review by dblock — APPROVED** [review]

**Pull request merged**